xain_fl/coordinator/coordinator.py

```diff
@@ -100,7 +96,7 @@ def __init__(  # pylint: disable=too-many-arguments
         num_rounds: int = 1,
         minimum_participants_in_round: int = 1,
         fraction_of_participants: float = 1.0,
-        weights: Optional[List[ndarray]] = None,
+        weights: ndarray = np.empty(shape=(0,)),
```

Comment:
it's not so obvious to me what `shape=(0,)` means. perhaps a short explanatory comment?

Reply:
basically it's a 1-dimensional array, but it has no coordinates. this is probably the closest we can get to an empty vector without using `None`. this is somehow needed, since `None` is not available as a value for a protobuf field.

it's not ambiguous, since every proper model should have at least one weight. also we can use numpy utility via `np.empty(shape=(0,)).size` which evaluates to `0`. and this empty vector is serializable with our xain-proto lib, which doesn't work for example for `np.array(None, dtype=object)` which would be another way to represent `None` as a value for a protobuf message.

Reply:
makes sense, if not totally obvious!